installer/bundle/bundle_skel.sh

```diff
@@ -1457,14 +1462,16 @@ case "$installMode" in
         ;;
 
     U)
+        install_tst="yes" # set variable to install tst upon exit
+
         # Install OMI
         shouldInstall_omi
         pkg_upd ${OMI_PKG} omi $?
         OMI_EXIT_STATUS=$?
-	${OMI_SERVICE} reload
-	temp_status=$?
+	    ${OMI_SERVICE} reload
```

Comment:
nit this section seems all weird indent

Reply:
It looks like there were some tabs used instead of spaces? Sorry I thought I was fixing the weird indentation on my end. Changed all the tabs in that area to spaces for consistency:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/1fcd4dd0d1cef4e18b71d295a048a3e4b7412dc9/installer/bundle/bundle_skel.sh#L1469-L1479